tests/lean/run/2389.lean

```diff
@@ -0,0 +1,35 @@
+/-!
+# Test verifying that nested predicates don't trigger the generation of `below` lemmas
+Since the case of nested predicates is not currently handled by `mkBelow`,
+trying to generate `Bar.below` triggers an error upon defining the inductive type.
```

Comment:
Also, reference to `mkBelow` should name the file `src/Lean/Meta/IndPredBelow.lean` to help someone find the relevant code.